### DISCUSSION TITLE
地下鼠成搜救機器人靈感，中國研發 SQuRo 機器鼠

### DISCUSSION THREAD

**Discussion by yi-1111** [1101-GS4719-C/Python-Community-GS4719-C#571] [科技新知]
![SQuRo-624x288](https://user-images.githubusercontent.com/100227107/165467037-7fa8cddb-26cc-42f8-b991-e84a0289e0df.png)

已知現在有機器搜救犬、搜救機器蛇，可以在各種狹窄裂縫與不平坦地形穿梭游移，但其實老鼠也很擅長在上述地形活動，中國科學團隊就以老鼠為靈感，研發出新型機器鼠 SQuRo。
機器鼠靈巧到可以擠過窄至 9 公分的不規則通道，走過高 3 公分的障礙物，並能爬上 15-度斜角。內建的機上微處理器控制機器鼠的任何動作，根據檢測到的地面反作用力，切換四種基本運動模式。

團隊希望之後進一步最佳化技術，SQuRo 可以應用在災難現場尋找倖存者或著派去難以到達的地區進行檢查等任務。
https://technews.tw/2022/04/23/squro-robotic-rat/

**Comment by HuanYuuuuuu** [comment]
這個科技好酷! 感謝你的分享!

**Comment by Ku-Hao-Chen** [comment]
謝謝分享，原來有這種東西。

**Comment by d26488399** [comment]
謝謝分享，好酷的機器

**Comment by cordydeng** [comment]
救援最需要的就是時間，以往的搜救機器人沒辦法在時間上取得優勢，這個技術一定可以拯救到需要幫助的人

**Comment by lx3211** [comment]
不知道以後會不會有變形機器人，融合蛇、鼠、犬為一體，每種動物的優點都可以有

**Comment by Lai27330** [comment]
對於搜救和地形探勘感覺很有幫助，而且也很像偵探片中的蒐集情報的間諜道具。

**Comment by HsuChen88** [comment]
以現有的動物為靈感，加以改良的話應該能更佳地運用

**Comment by HaoFish123** [comment]
感謝分享，其實以前有聽過蛇型的機器，不過還是希望這些機器可以真的派上用場。

**Comment by treeann** [comment]
地下屬這個銘子也太帥了八

**Comment by yucheng1028** [comment]
各種仿生機器人越來越多了

**Comment by alexthebigpp** [comment]
這種機器鼠在台灣這種地震很多的地方大有可為。

**Comment by yenwei12138** [comment]
這比人形機器人可愛多了~但這材質感覺很不耐撞耐壓?

**Comment by etin2468** [comment]
這對常發生天然災害的國家很有幫助ㄟ
